package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "zq",
```

Comment:
the trailing comma isn't valid in json...

Reply:
🤦‍♂